integreat_cms/cms/models/users/organization.py

```diff
@@ -61,6 +61,23 @@ def get_repr(self):
         """
         return f"<Organization (id: {self.id}, slug: {self.slug}, region: {self.region.slug})>"
 
+    @property
+    def num_pages(self):
+        """
+
+        :return: the current number of maintained pages of an organization object
+        :rtype: int
+        """
+        return self.pages.count
```

Comment:
```suggestion
        return self.pages.count()
```
In templates, this works since callables are automatically called, but if this property is used e.g. in a view, this would return only the method `count` and not the actual value.

Reply:
My bad. I did not notice it since I just used it in the template. :man_facepalming: